grails-app/migrations/changelog-2.0.2.groovy

```diff
@@ -0,0 +1,7 @@
+databaseChangeLog = {
+	changeSet(author: "cmdcolin (generated)", id: "1454711582784-1") {
+        createIndex( indexName: "feature_event_s123", tableName: "feature_event", unique: "false" ) {
```

Comment:
the name here should correspond to the name in the FeatureEvent class, which I believe is ```feature_uniqueName```

I also think you need something like ```indexExists```:  http://www.liquibase.org/documentation/preconditions.html